dist/ventus.js

```diff
@@ -1201,7 +1203,9 @@ define('ventus/wm/window', [
                     x: event.pageX,
                     y: event.pageY
                 });
-                this.el.addClass('move');
```

Comment:
These classes give semantic information about the window state that can be used for other styling purposes, not only animating. I would rather add a new class "animated" to the window when the `animations`  option is enabled, and change the CSS selectors to apply the animation only to the windows with that class.

The JS code will remain the same, we will only add:
```js
if (this.animations) {
  this.el.addClass('animated');
}
```

Reply:
I understand, let me fix this...

Reply:
I have fixed your concerns, can you have a look at it again? :dancer: